content/english/linux-basics/activity-8.md

```diff
@@ -28,20 +30,21 @@ ls -la
 The `-a` flag gives all entries including hidden files.
 {{% /notice %}}
 
-*Did you find any new files? Put that in your report!*
+*Did you find any new files? Take notes of the changes!*
 
 ### Compare the secret-message.txt file to the backup file
 
 `diff` is a command used to compare two files to get what has been added, changed, and deleted to the file.  
-The format is: `diff -u [filename] [Filename of Comparison File]`
+The format is: `diff [filename] [Filename of Comparison File]`
 
 {{% notice info %}}
-Fun Fact: The `-u` flag puts the output in unified mode. You can change the `u` to a `c` to put the output in context mode. You can also remove the flag altogether to get the output without any formatting. 
+Fun Fact: Running `diff --help` will output information on how to use the `diff` tool, including what options can be used with the tool. 
 {{% /notice %}}
 
 Check the difference between the two files. How many differences did you find? Put it in the file you made!
 
 {{% notice tip %}}
+
 ### A better way to see the differences
 
 `vimdiff` is a command that opens a window with the files in separate tabs highlighting the differences.
```

Comment:
Are we able to get a screenshot for this?

Reply:
Oh I must have missed this one. Should I try to get it?